dbt/tests/generic/test_accepted_range.sql

```diff
@@ -0,0 +1,44 @@
+-- Override `dbt_utils.accepted_range` generic so that it can return extra
+-- columns for debugging
+{% test accepted_range(
+    model,
+    column_name,
+    min_value=none,
+    max_value=none,
+    inclusive=true,
+    select_columns=[]
+) %}
+
+    {%- set columns_csv = select_columns | join(", ") %}
+
+    with
+        meet_condition as (
+            select {{ column_name }} {%- if columns_csv %}, {{ columns_csv }}{% endif %}
```

Comment:
Now that I've refactored our generic tests to accept a `select_columns` param, I'm wondering whether we should standardize on `additional_select_columns` instead... these tests are all written to select `column_name` by default if it exists in the test, so perhaps `additional_select_columns` makes it clearer what the param does? (The semantics is important since if a column is included twice in the `SELECT` statement, Athena will raise a somewhat confusing error.) What do you think is the clearer interface?

Reply:
I do think `additional_select_columns` is a bit clearer for column-level tests.

Reply:
I think that makes sense, done in f0a0586bc123834ae3d615668bf96cb56341493e! The only gray area is `expression_is_true`, which is sometimes a column-level test and sometimes a table-level test. Since we use it much more often as a table-level test, and since the column-level versions are probably ripe for refactoring for other generic test functions anyway, I went ahead and kept its param as `select_columns`, even though there's one or two instances where we use it on a column-level test where we already select `column_name` by default.